CDF Election Process.md

```diff
@@ -0,0 +1,133 @@
+
+
+### Who Can Nominate
+
+Nominations are dependent on the position up for election. Please see the list below  for information on who is eligible to vote and nominate.
+
+
+ #### Governing Board Chair 
+
+* Nominations: Governing Board 
+* Voting: Governing Board
+
+
+#### General Member Representative 1+2 
+
+* Nominations: End User and General Members
+* Voting: End User and General Members
+
+
+#### Committer/Contributor Representative 1+2
+
+* Nominations: anyone in CDF Community
+* Voting: Governing Board
+
+
+#### Treasurer/ Budget Committee Chair
+
+* Nominations: Governing Board 
+* Voting: Governing Board
+
+
+#### TOC Chair
+
+* Nominations: CDF TOC Voting Members
+* Voting: CDF TOC Voting Members
+
+
+#### Project Representative 1-4 (elections managed by projects)
+
+* Nominations: Each CDF project will elect their own project representative
+
+
+#### End User Representative 1+ 2
+
+* Nominations: End User and General Members
+* Voting: End User and General Members
+
+
+#### TOC GB Representative 1-3
+
+* Nominations: TOC, GB 
+* Voting: Governing Board
+
+
+#### Outreach Committee Chair
+
+* Nominations: Anyone representing a member company may be nominated
+* Voting: Member company representatives and project appointed representatives
+
+
+#### Elections Officer
+
+* Nominations: Anyone currently holding a voting positions on any CDF committee may be nominated
+* Voting: Anyone currently holding a voting positions on any CDF committee may vote
+
+
+#### Awards Officer (open to suggestions)
+
+* Nominations
+* Voting
+
+
+## Voting
+
+Nominees will be announced on the [CDF website](https://cd.foundation/news/) the day prior to the start of the Voting Phase and the links to the announcement will be shared on corresponding Slack Channels and maillists. Ballots will be distributed via Google Form links.
+
+
+### How to Vote
+
+<span style="text-decoration:underline;">The Voting Phase is 1 week</span>
+
+At the completion of the Nomination Period eligible voters will be sent a ballot link. These links are unique to a voting class and should not be shared with anyone. Each member of the voting class is entitled to one vote per open position. 
+
+Voters should review the bios and statements of intent for each candidate before making their selection.
+
+Voters should keep their selections private to ensure the integrity of the election is maintained.
+
+
+### Who can Vote
+
+Voting classes differ depending on the charter statements of the open position. Please select the position you would like to vote on from the list above for information on who is eligible to vote and nominate.
+
+
+## Announcing the Winner
+
+Winners will be determined by a majority vote of the established voting class and announced the Monday following the Election Phase via Email, Slack, Social, and the [CDF Website](https://cd.foundation/news/).
```

Comment:
We need to add another step here to make sure the elected individuals are added to corresponding groups such as CDF TOC Team on GitHub for the case of TOC elections and GB lists for the chair.